### PR TITLE
Redirect_https_pattern

### PR DESCRIPTION
**Pull request by SimonGivan**
Updating Redirected sites patterns to match

### CODE REVIEW

File: frontdoor.tf [hunk 1]
```diff
@@ -168,7 +168,7 @@ resource "azurerm_frontdoor" "main" {
     content {
       name               = "${host.value["name"]}HttpsRedirect"
       accepted_protocols = ["Http"]
-      patterns_to_match  = ["/*"]
+      patterns_to_match  = lookup(host.value, "https_redirect_url_patterns", ["/*"])
```
Comment (timja) [inline frontdoor.tf:171]: why?

Comment (SimonGivan) [reply frontdoor.tf:171]: To allow a redirect pattern to match such as 
image

Comment (timja) [reply frontdoor.tf:171]: this functionality is only for redirecting http to https, do you really want only that url to go to https?

Comment (SimonGivan) [reply frontdoor.tf:171]: That one for now, half expecting more to be added

**Pull request closed without merging**